Foundation/FileManager+Win32.swift

```diff
@@ -299,9 +299,84 @@ extension FileManager {
     }
 
     internal func _destinationOfSymbolicLink(atPath path: String) throws -> String {
-        return try _canonicalizedPath(toFileAtPath: path)
+        let faAttributes = try windowsFileAttributes(atPath: path)
```

Comment:
Should the error here not be propagated?

Reply:
windowsFileAttributes throws if the file is not found which is what we also want to do.